bin/Powheg/production/2017/13TeV/st_tch_4f_ckm_13TeV/st_tch_4f_ckm_antitop_13TeV_UL.input

```diff
@@ -49,26 +49,12 @@ xupbound 2d0      ! increase upper bound for radiation generation
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 ! PROCESS SPECIFIC PARAMETERS
 
-tdec/elbranching 0.108  ! W electronic branching fraction
-tdec/sin2cabibbo 0.051
-topwidth 1.33
-
```

Comment:
all these lines here and below are the one where you get a warning from powheg ?

Reply:
Indeed:
```
 powheginput WARNING: unused variable tdec/elbranching    
 powheginput WARNING: unused variable tdec/sin2cabibbo    
 powheginput WARNING: unused variable topwidth            
 powheginput WARNING: unused variable wwidth              
 powheginput WARNING: unused variable lhfm/emass          
 powheginput WARNING: unused variable lhfm/mumass         
 powheginput WARNING: unused variable lhfm/taumass        
 powheginput WARNING: unused variable lhfm/cmass          
 powheginput WARNING: unused variable lhfm/bmass          
 powheginput WARNING: unused variable tdec/emass          
 powheginput WARNING: unused variable tdec/mumass         
 powheginput WARNING: unused variable tdec/taumass
```